management-center/Dockerfile

```diff
@@ -1,11 +1,19 @@
 FROM java:7-jre
 ENV HZ_VERSION 3.8
 ENV HZ_HOME /opt/hazelcast/
+# Persistent VOLUME definitions
+ENV MANCENTER_DATA /data
+VOLUME ["/data"]
```

Comment:
I think we should try to find a general naming set for this directories. In the new openshift docker file it is called _external_ (https://github.com/hazelcast/hazelcast-docker/blob/master/hazelcast-openshift-rhel/Dockerfile#L5) and is automatically added to the classpath.